config/flags/flags.go

```diff
@@ -331,4 +331,10 @@ var (
 		Usage: "Password to unlock validator private key",
 		Value: "",
 	}
+	
+	// Consensus
+	SuppressFramePanicFlag = cli.BoolFlag{
+		Name:  "suppress-frame-panic",
```

Comment:
It looks like the convention in this list of flags is to use a naming schema like `<component>.<property>` to group flags. The property is all-lower-case without spaces. According to this, this flag should be named something like `lachesis.suppressframepanic` or `lachesis.ignoreframemissmatch`. Due to the long name I would probably take `lachesis.ignore-frame-miss-match`.
Wdyt?